lib/perm/is-own-invitation.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+function isOwnInvitation (args, cb) {
+  var seneca = this;
+  var userId, invitation;
+  if (args.params.invitation) invitation = args.params.invitation;
```

Comment:
Why bother with the if? The result is the same with or without it. `invitation` would be undefined if `args.params.invitation` is undefined.